sql/moz-fx-data-shared-prod/static/fxa_amplitude_export_users_daily/metadata.yaml

```diff
@@ -0,0 +1,6 @@
+friendly_name: |-
+  Static - Fxa Amplitude Export Users Da
```

Comment:
Should these names be cut off?

Reply:
no absolutely not ... will have a look at how this happened, thanks for catching it

Reply:
ah the script expected a `_v1` at the end but this is obviously not the case everywhere
will fix that

Reply:
Thanks @lelilia for fixing this. 🙇